ouroboros-consensus-cardano/changelog.d/20240430_233346_Lucsanszky_integration_8_11.md

```diff
@@ -0,0 +1,20 @@
+### Patch
+
+- Update `SupportsTwoPhaseValidation` instance for `Conway`:
+  `ConwayUtxowFailure` was changed upstream
+
+### Non-Breaking
+
+- Update ledger packages:
+  - `cardano-ledger-alonzo`: `^>=1.8`
+  - `cardano-ledger-babbage`: `^>=1.8`
+  - `cardano-ledger-conway`: `^>=1.14`
+  - `cardano-ledger-core`: `^>=1.12`
+  - `cardano-ledger-mary`: `^>=1.6`
+  - `cardano-ledger-shelley`: `^>=1.11`
+
+### Breaking
+
+- Add `c ~ StandardCrypto` constraint to:
+  - `mkSimpleTestProtocolInfo`
+  - `mkTestProtocolInfo`
```

Comment:
Unless I'm mistaken, these are changes in the test libraries. We don't record changes to those in the changelog.

Reply:
Yes, you are right. Those are only related to test libraries, so if you don't track those then I can remove the `Breaking` part of the fragment.

Reply:
Yes, please do so